src/Tizen.Multimedia.Remoting/WebRTC/WebRTCDataChannel.cs

```diff
@@ -113,6 +113,57 @@ private IntPtr Handle
         /// <since_tizen> 9 </since_tizen>
         public string Label { get; }
 
+        /// <summary>
+        /// Gets the amount of buffered data.
+        /// </summary>
+        /// <value>The buffered amount in bytes.</value>
+        /// <since_tizen> 10 </since_tizen>
+        public uint BufferedAmount
+        {
+            get
+            {
+                ValidateNotDisposed();
+
+                NativeDataChannel.GetBufferedAmount(Handle, out uint amount).
+                    ThrowIfFailed("Failed to get buffered amount");
+
+                return amount;
+            }
+        }
+
+        private uint? _bufferThreshold;
+        /// <summary>
+        /// Gets or sets the threshold of data channel buffered amount.<br/>
+        /// If the amount of buffered data is lower than threshold value, <see cref="BufferedAmountLow"/> will be occurred.<br/>
+        /// The default value is 0. and if threshold is 0, <see cref="BufferedAmountLow"/> is not occurred.
```

Comment:
after dot "." new sentence should be started with capital letter,  "be occurred" should replaced by "be raised" or sth like that.
```suggestion
        /// The default value is 0, which means <see cref="BufferedAmountLow"/> is disabled and will not be raised.
```

Reply:
Okay. I'll fix it.